### DISCUSSION TITLE
Release date for 0.3.16

### DISCUSSION THREAD

**Discussion by mitulshah-suse** [Mirantis/cri-dockerd#411] [General]
When the release supposed to come out? We need some of the fixes that have gone in.

**Comment by afbjorklund** [comment]
It might also need to downgrade docker to a supported release, and update the packaging distros (24.04/el9)...

**Comment by afbjorklund** [comment]
https://github.com/Mirantis/cri-dockerd/releases/tag/v0.3.16
